core/homophones/homophones_open.talon

```diff
@@ -1,9 +1,7 @@
 tag: user.homophones_open
 -
-choose <number_small>:
-    result = user.homophones_select(number_small)
-    insert(result)
-    user.homophones_hide()
+tag(): user.choose_selector
+
```

Comment:
what is this tag for?

Reply:
I was confused about how the overrides work. Seems unnecessary now.